src/Lib/Items/CanvasImage.vala

```diff
@@ -91,11 +91,10 @@ public class Akira.Lib.Items.CanvasImage : Goo.CanvasImage, Akira.Lib.Items.Canv
                 // Change the locked attribute after the size has been defined to let
                 // the Size component properly calculate the correct size ratio.
                 size.locked = true;
-
                 // Reset the size to a 2px initial value after the size ratio was properly defined
                 // in order to allow the user to decide the initial image size. We use 2px in order
                 // to avoid issues when dividing by the ratio in case of narrow images.
-                size.width = 2;
+                // size.width = 2;
```

Comment:
You can't comment this out.
Please, read the comment above this declaration to understand why we're doing this.
If this interferes with the dropping of images, you need to find a smarter solution to properly size dropped images.